### PR TITLE
Adding support for pin slack and capacitance to Python APIs

### PR DESCRIPTION
**Pull request by VidyaChhabria**

### CODE REVIEW

File: include/ord/Timing.h [hunk 1]
```diff
@@ -104,6 +107,7 @@ class Timing
   bool isEndpoint_(sta::Pin* sta_pin);
   float getPinSlew_(sta::Pin* sta_pin, MinMax minmax);
   float getPinArrival_(sta::Pin* sta_pin, RiseFall rf, MinMax minmax);
+  float getPinSlack_(sta::Pin* sta_pin, RiseFall rf, MinMax minmax);
```
Comment (maliberty) [inline include/ord/Timing.h:110]: Why is there an underscore in the name?

File: include/ord/Design.h [hunk 2]
```diff
@@ -172,6 +172,7 @@ class Design
     Max
   };
   float getNetCap(odb::dbNet* net, sta::Corner* corner, MinMax minmax);
+  float getPortCap(odb::dbITerm* pin, sta::Corner* corner, MinMax minmax);
```
Comment (maliberty) [inline include/ord/Design.h:175]: I would rather see APIs that require sta objects moved to Timing.h so that Design is more about the netlist (obd).

**Review by maliberty — CHANGES_REQUESTED** [review]

File: include/ord/Timing.h (hunk 1 above)
Comment (VidyaChhabria) [reply include/ord/Timing.h:118]: This was just a convention that I had followed, indicating that the function is a private version of the user-facing public API. These private functions can access internal objects like STA pins. If there is a better coding style, I can change it; please let me know what would be appropriate.

Comment (maliberty) [reply include/ord/Timing.h:118]: I would just drop the underscore as the overload resolution is pretty clear.  If you prefer a different name then I would replace underscore with Priv or Impl.

**Review by maliberty — APPROVED** [review]

**Pull request merged**